lib/middlewares/methodnoallow.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
 const methods = require('methods');
-
+const METHODS_NOT_ALLOWED = [ 'trace', 'track' ];
 const safeHttpMethodsMap = {};
+
 for (const method of methods) {
-  if ([ 'trace', 'track', 'options' ].indexOf(method) === -1) {
+  if (METHODS_NOT_ALLOWED.indexOf(method) === -1) {
```

Comment:
use `includes` because we only support node >= 8

Reply:
OK